Release.toml

```diff
@@ -140,5 +140,8 @@ version = "1.9.1"
     "migrate_v1.9.0_kubelet-no-daemon-reload.lz4",
```

Comment:
Nit: for the lack of a better place for this comment

The subject for this commit is missing a "subsystem", in other commits `netdog` was used.